go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/Azure/kubelogin
 
 // NOTE: kubelogin follows the same support policy as Go, which supports the last two major versions.
-go 1.21
+go 1.21.0
```

Comment:
should it be one of the most recent 1.21 instead? its currently built using 1.21.9

Reply:
updated